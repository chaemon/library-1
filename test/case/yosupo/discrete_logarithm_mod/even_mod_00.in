100
156648746 154396384 916122984
318065816 264440383 864460180
648104885 725454492 942496936
543030428 663661240 851230462
108201616 428505484 970970328
322953840 119811886 962418634
306647414 549862302 840030280
455464665 307270851 885397982
722629343 875317523 895196834
244419357 781214570 986135330
304620505 633084214 924905086
567530398 381873603 874996602
7218487 144376610 846315838
780877708 589558486 977171796
721407802 553010676 824595224
381588618 132249152 831569230
349655325 541586881 937804508
459480021 458194371 827906968
805750859 225659487 880876800
541627396 189845338 960097458
705924883 701326768 816362696
153907111 153703898 984724212
610379669 633051911 887367818
356035805 857066210 891070418
911192541 522937781 969592286
175671834 3720977 996860048
537811758 487222100 838548564
514312094 134561869 898124622
532291996 651125279 848951862
599149591 243222914 926148816
252728018 259733480 840997394
251320507 149124068 810327218
331487169 269144629 823882776
248342016 406539487 910438838
755591984 346209706 881054984
192576280 339470264 926230902
674221023 82572049 982623654
293935164 823871256 859378272
552250085 602446372 847070754
36819590 716162336 916008300
877980488 234609344 896641440
96912084 154071710 862231748
241584076 888321423 897906342
444486812 306061338 939683386
718697235 185635311 806600512
175477372 385632116 815576570
281221795 749035961 948228420
403681059 477587442 939150234
344958618 247221529 815718814
390143235 215677770 977599652
421663553 829251359 939231894
477181978 395005915 942634196
819349760 65949542 940600570
779931867 351133755 887966510
60835260 97550049 911531796
568484556 259645749 828695550
633917839 25145201 834352302
236424237 62605193 827471396
550125446 754369251 847379218
503316297 263982839 995583390
583095008 633551381 970671564
28565500 11547032 965238520
794174955 544258177 819797532
105432268 44562564 940574008
205744622 56560140 908143460
788812018 759517420 859483884
786197015 332668941 930288966
354325646 78199529 824897880
792547135 344979556 855656246
854248606 694655264 939318150
675361364 540932136 997689512
720653471 638568586 971490884
199659791 923181961 948744334
568202699 711859964 941963352
835360370 264697884 951228172
729753828 368415357 966053296
789440908 599511875 922817312
56060135 326810208 812810004
315946986 784951414 842952786
37285994 754591963 839920916
738593187 784935039 978553450
778665738 332058127 894000364
24400815 386891485 982734994
182407666 636003294 997487606
163815973 457948520 875404476
638332658 600592575 995042590
156722719 359729359 848785864
235226467 568069676 847039212
510134242 577467057 828623570
752028766 164889532 901957610
710088559 522973050 966125608
850448894 695807095 998888456
765043644 5572719 947066586
370336434 180999313 956715762
787390629 687572860 981638682
791821786 162447801 812986060
767002401 673284106 830273154
155385500 806266306 847209298
683743955 593913331 971018950
367144078 404043218 969772524
