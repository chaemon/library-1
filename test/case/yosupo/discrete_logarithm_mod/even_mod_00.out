1618984
-1
-1
-1
-1
13950756
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
64789115
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
66935973
-1
-1
-1
-1
-1
-1
-1
-1
-1
84815970
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
